data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have disabled the Korath ship that was left over from the raid on <planet>. You can now return there to collect your payment."
+	on complete
+		payment 500000
+		dialog `Back in the spaceport you are met by the prefect who thanks you for disabling it and hands you a credstick for <payment>. "Oh, and if you managed to plunder anything, don't forget to drop it off in the outfitters so the researchers can look at it.`
+
+mission "Remnant: Bounty"
+	job
+	repeat
+	name "Remnant bounty"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Salvage 4: done"
+		random < 60
+	npc kill
+		government Korath
+		personality coward target uninterested marked waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
+	on complete
+		payment 200000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ship <npc>, and gives you the agreed-upon payment of <payment>."
+
+		
+		
+mission "Remnant: Salvage 5"
+	name "Salvage Delivery"
+	description `Deliver some high value salvage to Caelian`
+	cargo "Korath Salvage" 20
+	blocked `This delivery request requires 20 tons of cargo space.`
+	source "Viminal"
+	destination "Caelian"
+	to offer
+		has "license: Remnant"
+		has "Remnant: Salvage 4: done"
+	on offer
+		conversation
+			`As you stroll across the tarmac under the great dome you spot Tali heading your way with a flatbed tractor filled with components in tow.`
+			`	"<first>! I'm glad I ran into you. The last raid gave us some particularly good pieces of equipment to study. Could you take these to the weapons laboratory on Caelian?"`
+			choice
+				`	"I could do that"`
+					goto DeliverSalvage
+				`	Not interested.`
+					decline
+			label "DeliverSalvage"
+			`	Tali quickly helps load the cargo onboard your ship. It doesn't appear to be actual whole items anymore, but rather pieces thereof.`
+			`	"They will be expecting you, but you might have to pry them out of their lab to get some help unloading. They can be a bit single-minded when they have something interesting to work on."`
+				accept
+	
+	on complete
+		payment 50000
+		dialog `On Caelian you are able to find the appropriate research warehouse just off the main landing pad. After a few minutes a lab tech disentangles  herself from a project and brings a wagon and camel team out to the ship to unload the salvage. It strikes you as a bit incongruous that a lab tech is using a graviton repulsion lifter to load cargo from your ship into a wagon pulled by a camel. "If you can, I should have some things to send back to Tali if you could meet me in the cafeteria this afternoon."`
```

Comment:
Comma after "Caelian" (or maybe remove "On Caelian"), comma after "few minutes", and remove the extra space between "disentangles" and "herself"

Reply:
Opted to leave "On Caelian" for now. I like it, but I'm not set on it. Fixed the rest.